code/modules/mob/living/simple_animal/friendly/drone/_drone.dm

```diff
@@ -95,6 +96,7 @@
 	access_card = new /obj/item/card/id(src)
 	var/datum/job/captain/C = new /datum/job/captain
 	access_card.access = C.get_access()
+	radio = new /obj/item/radio/headset/silicon/ai(src)
```

Comment:
you'll need to delete the radio when they die

Reply:
Yeah I kinda forgot I needed to do that
I'll get around to it soon